gulp/tasks/clean.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var del = require( 'del' );
+var config = require( '../config' ).clean;
```

Comment:
name `var configClean` or something to differentiate from `var config = require( '../config' );`